Add prow-addons-ctrl job and slack token secret information and checks (#595)
kyma-project/test-infra: docs/prow/prow-installation-on-forks.md

 - **sa-gcr-push-kyma-project** with the role that allows the account to push images to Google Container Repository in the `kyma-project` GCP project:
    - Storage Admin (`roles/storage.admin`)
 - **kyma-bot-npm-token** which is a token for publishing npm packages

>  - **sap-slack-bot-token** which is a token for publishing messages to SAP CX workspace. More information can bo found [here](https://api.slack.com/docs/token-types#bot).

Reviewer: ```suggestion
 - **sap-slack-bot-token** which is a token for publishing messages to the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot).
```
